bigtable/google/cloud/bigtable/table.py

```diff
@@ -617,10 +628,16 @@ def _create_row_request(table_name, row_key=None, start_key=None, end_key=None,
     message = data_messages_v2_pb2.ReadRowsRequest(**request_kwargs)
 
     if row_key is not None:
-        message.rows.row_keys.append(_to_bytes(row_key))
+        row_set = RowSet()
+        row_set.add_row_key(row_key)
+
+    if start_key is not None or end_key is not None:
+        row_set = RowSet()
+        row_set.add_row_range(RowRange(start_key, end_key,
+                                       end_inclusive=end_inclusive))
```

Comment:
Do we need to add a `start_inclusive` parameter here?

Reply:
I just added row_set as an extra parameter here in "_create_row_request" method. I thought like, in later releases, we can remove start_key, end_key and end_inclusive parameters, as that range can be included in the row_set parameter.
Let me know if i should add "start_inclusive" parameter in _create_row_request method.